File: SDKs/Aspose.Pdf-Cloud-SDK-for-.NET/src/Com/Aspose/PDF/Model/Field.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.PDF.Model {
  public class Field {
    public string Name { get; set; }

    public int Type { get; set; }

    public List<string> Values { get; set; }

    public List<int?> SelectedItems { get; set; }

    public Rectangle Rect { get; set; }

    public List<Link> Links { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Field {\n");
      sb.Append("  Name: ").Append(Name).Append("\n");
      sb.Append("  Type: ").Append(Type).Append("\n");
      sb.Append("  Values: ").Append(Values).Append("\n");
      sb.Append("  SelectedItems: ").Append(SelectedItems).Append("\n");
      sb.Append("  Rect: ").Append(Rect).Append("\n");
      sb.Append("  Links: ").Append(Links).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: SDKs/Aspose.Pdf-Cloud-SDK-for-Android/Aspose-PDF-Cloud-SDK-for-Android/src/main/java/com/aspose/pdf/model/Field.java
package com.aspose.pdf.model;

import java.util.*;
import com.aspose.pdf.model.FieldType;
import com.aspose.pdf.model.Rectangle;
public class Field {
  private String Name = null;
  private Integer Type = null;
  private List<String> Values = new ArrayList<String>();
  private List<Integer> SelectedItems = new ArrayList<Integer>();
  private Rectangle Rect = null;
  private List<Link> Links = new ArrayList<Link>();
  /**
	 * getName
	 * Gets String
	 * @return Name
	 */
  public String getName() {
    return Name;
  }

	/**
	 * setName
	 * Sets String
	 * @param Name String
	 */
  public void setName(String Name) {
    this.Name = Name;
  }

  /**
	 * getType
	 * Gets FieldType
	 * @return Type
	 */
  public Integer getType() {
    return Type;
  }

	/**
	 * setType
	 * Sets FieldType
	 * @param Type FieldType
	 */
  public void setType(Integer Type) {
    this.Type = Type;
  }

  /**
	 * getValues
	 * Gets List<String>
	 * @return Values
	 */
  public List<String> getValues() {
    return Values;
  }

	/**
	 * setValues
	 * Sets List<String>
	 * @param Values List<String>
	 */
  public void setValues(List<String> Values) {
    this.Values = Values;
  }

  /**
	 * getSelectedItems
	 * Gets List<Integer>
	 * @return SelectedItems
	 */
  public List<Integer> getSelectedItems() {
    return SelectedItems;
  }

	/**
	 * setSelectedItems
	 * Sets List<Integer>
	 * @param SelectedItems List<Integer>
	 */
  public void setSelectedItems(List<Integer> SelectedItems) {
    this.SelectedItems = SelectedItems;
  }

  /**
	 * getRect
	 * Gets Rectangle
	 * @return Rect
	 */
  public Rectangle getRect() {
    return Rect;
  }

	/**
	 * setRect
	 * Sets Rectangle
	 * @param Rect Rectangle
	 */
  public void setRect(Rectangle Rect) {
    this.Rect = Rect;
  }

  /**
	 * getLinks
	 * Gets List<Link>
	 * @return Links
	 */
  public List<Link> getLinks() {
    return Links;
  }

	/**
	 * setLinks
	 * Sets List<Link>
	 * @param Links List<Link>
	 */
  public void setLinks(List<Link> Links) {
    this.Links = Links;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Field {\n");
    sb.append("  Name: ").append(Name).append("\n");
    sb.append("  Type: ").append(Type).append("\n");
    sb.append("  Values: ").append(Values).append("\n");
    sb.append("  SelectedItems: ").append(SelectedItems).append("\n");
    sb.append("  Rect: ").append(Rect).append("\n");
    sb.append("  Links: ").append(Links).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

